src/outcome_spaces/ordinal_patterns.jl

```diff
@@ -255,6 +255,8 @@ end
 function codify(est::OrdinalOutcomeSpace{m}, x) where m
     if x isa AbstractVector
         dataset = embed(x, m, est.τ)
+    elseif x isa AbstractStateSpaceSet && dimension(x) == 1
+        throw(ArgumentError("Convert your univariate time series to a subtype of `AbstractVector` to codify with ordinal patterns! A `StateSpaceSet` input is assumed to be already embedded in D = m >= 2 dimensional space."))
```

Comment:
While we are here, can we keep the total character length 92 or less per line of code? This seems to be well beyond it!

Reply:
Done.